src/main/resources/data.csv

```diff
@@ -0,0 +1,9 @@
+type,fruit,quantity
+b,banana,20
+b,apple,100
+s,banana,100
+p,banana,13
+r,apple,10
+p,apple,20
+p,banana,5
+s,banana,50
```

Comment:
I think m we don`t need those files. So remove it. Or, add a new line in the end of files)
(new line in the end of files it's really important, you can find info about it at internet)